### PR TITLE
Feature/#176 application footer

### LINKED ISSUE #176: Application Footer
We want a footer in the frontend which displays some important information.

- [ ] Build number
- [ ] Outcobra version number
Add whatever you want.

### PR DESCRIPTION
**Pull request by M4R1KU**
This closes #176  .

Summary of changes:
- ​added application footer

I made sure to:
- [x] Add and / or update tests
- [x] Run the code formatter (`CTRL + ALT + L`)
- [x] Update any documentation (javadoc, domain model, erd, ...)
- [x] Follow the contribution guidelines
- [x] Update CI scripts on Semaphore (changes to `auth0.yml` or `application.yml`)

Please review @jmesserli / cc: @outcobra/developers.

### CODE REVIEW

File: backend/build.gradle [hunk 1]
```diff
@@ -16,30 +16,32 @@ buildscript {
     dependencies {
         classpath($/org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion/$)
         classpath($/org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion/$)
-        classpath('gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:1.4.17')
```
Comment (needToRoll) [inline backend/build.gradle]: Do we really need this? And if so, what for?

**Comment by jmesserli** [comment]
If we don't have this, no git commit info is available at runtime.

needToRoll <notifications@github.com> schrieb am Sa., 25. März 2017, 17:02:

File: frontend/src/assets/config/dev.json [hunk 2]
```diff
@@ -6,6 +6,11 @@
         "name": "outcobra",
         "apiBase": "http://localhost:8080/api/",
         "authToken": true
+      },
+      {
+        "name": "outcobra_public",
```
Comment (jmesserli) [inline frontend/src/assets/config/dev.json]: What's this about?

Comment (M4R1KU) [inline frontend/src/assets/config/dev.json]: For the info endpoint and other endpoints that do not need authentication.

Comment (jmesserli) [inline frontend/src/assets/config/dev.json]: Oh I see thanks.

**Pull request merged**